src/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/Base64Transforms.cs

```diff
@@ -0,0 +1,294 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// This file contains two ICryptoTransforms: ToBase64Transform and FromBase64Transform
+// they may be attached to a CryptoStream in either read or write mode
+
+namespace System.Security.Cryptography
+{
+    using System;
+    using Text;
```

Comment:
```using System``` can be removed, and the ```using Text``` should be moved out of the namespace and expanded accordingly.